src/simulations/set_generator.py

```diff
@@ -51,6 +53,57 @@ def __iter__(self):
     for _ in range(self.num_sets):
       yield self.set_size
 
+def _choice_fast(n, m, random_state=np.random.RandomState()):
```

Comment:
I think this would do well in the common directory instead of in the set_generator.  I can't immediately think of anything else using it, but you never know what the future might hold.  At the least I don't think you need to worry about it being private: what made you decide to make it private?

Reply:
Oh, I didn't even realize there was a common directory! That probably would be a better place for it

Reply:
38fadaa